lib/main_common.pm

```diff
@@ -2435,9 +2435,81 @@ sub load_host_installation_modules {
     loadtest "installation/reboot_after_installation";
 }
 
+sub set_mu_virt_vars {
+    # Set UPDATE_PACKAGE based on BUILD(format example, BUILD=:33310:dtb-armv7l)
+    my $BUILD = get_var('BUILD', '');
+    $BUILD =~ /^:(\d+):([^:]+)$/im;
+
+    die "BUILD value is $BUILD, but does not match required format." if (!$2);
+
+    my $_pkg = $2;
+    my $_update_package = '';
+    if ($_pkg =~ /qemu|xen|virt-manager|libguestfs|libslirp|open-vm-tools/) {
+        $_update_package = $_pkg;
+    } elsif ($_pkg =~ /libvirt/) {
+        $_update_package = 'libvirt-client';
+    } else {
+        $_update_package = 'kernel-default';
+    }
+
+    set_var('UPDATE_PACKAGE', $_update_package);
+    diag("BUILD is $BUILD, UPDATE_PACKAGE is set to " . get_var('UPDATE_PACKAGE', ''));
+
+    # Set PATCH_WITH_ZYPPER
+    set_var('PATCH_WITH_ZYPPER', 1) unless (check_var('PATCH_WITH_ZYPPER', 0));
+
+    # Set AUTOYAST
+    if (check_var('HOST_INSTALL_AUTOYAST', 1) or check_var('AUTOYAST', 1)) {
```

Comment:
Why need extra ```HOST_INSTALL_AUTOYAST``` ? I think ```AUTOYAST``` is enough.

Reply:
That's my understanding too. But `HOST_INSTALL_AUTOYAST` has been in qem settings for long. So keep its usage as it has been.